pep-0581.rst

```diff
@@ -69,29 +74,33 @@ not currently available on Roundup / bpo.
   systematic filtering of emails.
 
 - Additional privacy, such as offering the user a choice to hide an
-  email address, while still allowing communication with the user through @-mentions.
+  email address, while still allowing communication with the user
+  through @-mentions.
+
 
 Issues with Roundup / bpo
 -------------------------
 
 - Less than five people maintain bpo. Some of them are core developers.
 
-- It is in Mercurial. Without any CI available, it puts heavy burden on the few
-  existing maintainers in terms of reviewing, testing, and applying patches.
+- The upstream Roundup code is in Mercurial. Without any CI available,
```

Comment:
Personally, I don't see how this is a problem here. `pip install` supports installing from Mercurial repositories and its testing suite isn't that hard to run. Plus, as I said below, our Roundup instance can easily be moved to GitHub.